themes/default/content/docs/intro/concepts/inputs-outputs.md

```diff
@@ -651,6 +651,327 @@ variables:
 
 {{% /choosable %}}
 
+## Working with Outputs and JSON {#outputs-and-json}
+
+Often in the course of working with web technologies, you encounter JavaScript Object Notation (JSON) which is a popular specification for representing data. Pulumi provides first-class helper functions for deserializing JSON string outputs into your language's native objects and serializing your language's native objects to JSON string outputs.
+
+{{< chooser language "javascript,typescript,python,go,csharp" >}}
+
+{{% choosable language javascript %}}
+
+### Example Converting Outputs to JSON
```

Comment:
We shouldn't put each example under a header like this.
It pollutes the TOC on the right:
<img width="354" alt="image" src="https://user-images.githubusercontent.com/41730/228401730-f8a21e99-3318-46ff-a33e-b98c17284046.png">

It may be better to make these bold instead

```markdown
**Converting Outputs to JSON**
```

Reply:
unfortunately, we should not use bold in place of a heading; we can instead have an h3 of Examples which will show up in the table of contents, then you can use h4s from all the various examples and they wont populate in the table of contents if that's concerning to yall.

Reply:
ref: https://usability.yale.edu/web-accessibility/articles/headings#:~:text=Do%20not%20use%20bold%20instead,reader%20users%20will%20not%20benefit.